### PR TITLE
[Playwright] Fix broken Osteo tests

### PR DESCRIPTION
**Pull request by aweng98**
Broken Osteo tests because `getByRole('combobox').selectOption()` did not work for the `Select` elements. Created `osteo-fixture.ts` to handle entering site password, eliminated duplicated code in Osteo tests.

Ran several times on localhost to ensure tests can pass repeately. 

Screenshot 2023-02-21 at 5 45 43 PM

### CODE REVIEW

File: playwright-e2e/tests/osteo/osteo-regression-test.spec.ts [hunk 1]
```diff
@@ -536,9 +536,7 @@ test.fixme('Osteo self enroll @osteo', async ({ page }) => {
   await page.getByLabel('Enter age').click();
 
   await page.getByLabel('Enter age').fill('30');
-
-  // wait for country selection to drive state/province
-  await page.waitForResponse((response) => response.url().includes('/pepper/v1/user') && response.status() === 200);
```
Comment (aweng98) [inline playwright-e2e/tests/osteo/osteo-regression-test.spec.ts:541]: This request was not found.

File: playwright-e2e/pages/family-history.ts [hunk 2]
```diff
@@ -152,11 +153,17 @@ export class FamilyHistory extends BrainBasePage {
       } else if (selectorBase.indexOf('HALF_SIBLING') > 0) {
         sideOfFamilyClass = 'FAMILY_SIDE_Q_2';
       }
-      await this.page.locator(`.picklist-answer-${sideOfFamilyClass}`).getByRole('combobox').selectOption(p.sideOfFamily);
```
Comment (aweng98) [inline playwright-e2e/pages/family-history.ts:155]: `getByRole('combobox').selectOption` is not working for `Select` element.

File: playwright-e2e/fixtures/osteo-fixture.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { fixtureBase as base } from 'fixtures/fixture-base';
+import { fillSitePassword } from 'utils/test-utils';
+
+const { OSTEO_BASE_URL } = process.env;
+
+// Use this fixture in Osteo test
+const fixture = base.extend({
```
Comment (aweng98) [inline playwright-e2e/fixtures/osteo-fixture.ts:7]: Fixture used in Osteo test for entering site password.

**Review by Rinchan20 — APPROVED** [review]

**Comment by aweng98** [comment]
Thank you.

**Pull request merged**